mappings/net/minecraft/client/gui/widget/TextFieldWidget.mapping

```diff
@@ -49,7 +51,12 @@ CLASS cww net/minecraft/client/gui/widget/TextFieldWidget
 		ARG 1 cursor
 	METHOD g getCursor ()I
 	METHOD g setMaxLength (I)V
+		ARG 1 maximumLength
+	METHOD h setEditableColor (I)V
+		ARG 1 color
 	METHOD i isVisible ()Z
+	METHOD i setUneditableColor (I)V
+		ARG 1 color
```

Comment:
A setter for something you cannot edit? What exactly is `uneditableColor` and `editableColor`?

Reply:
the text color of the textfieldwidget 
For example, it uses the editableColor if it is editable.